.github/workflows/reusable-should-validate.yml

```
@@ -56,6 +54,7 @@ jobs:
         uses: dorny/paths-filter@v2
```

actionlint (1.7.7)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)